resources/views/tournaments/view.blade.php

```diff
@@ -256,6 +256,57 @@
                     </div>
                 </div>
             @endif
+            {{--Videos--}}
+            <div class="bracket">
+            @if ($tournament->concluded)
```

Comment:
Put the @if before the <div class="bracket">, otherwise you will get an empty bracket for non-concluded tournaments.